react_juce/core/TextView.h

```diff
@@ -38,21 +38,27 @@ namespace blueprint
         TextView() = default;
 
         //==============================================================================
-        /** Assembles a Font from the current node properties. */
-        juce::Font getFont()
+        /** Assembles a Font from properties. */
+        static juce::Font getFont(const juce::NamedValueSet &fontProps)
         {
-            float fontHeight = props.getWithDefault(fontSizeProp, 12.0f);
-            int textStyleFlags = props.getWithDefault(fontStyleProp, 0);
+            const float fontHeight = fontProps.getWithDefault(fontSizeProp, 12.0f);
+            const int textStyleFlags = fontProps.getWithDefault(fontStyleProp, 0);
 
             juce::Font f (fontHeight);
 
-            if (props.contains(fontFamilyProp))
-                f = juce::Font (props[fontFamilyProp], fontHeight, textStyleFlags);
+            if (fontProps.contains(fontFamilyProp))
+                f = juce::Font (fontProps[fontFamilyProp], fontHeight, textStyleFlags);
 
-            f.setExtraKerningFactor(props.getWithDefault(kerningFactorProp, 0.0));
+            f.setExtraKerningFactor(fontProps.getWithDefault(kerningFactorProp, 0.0));
             return f;
         }
 
+        /** Assembles a Font from the current node properties. */
+        juce::Font getFont()
```

Comment:
This should be const if possible `juce::Front getFont() const`

Reply:
OK!
Maybe I can use handle better next time :)
973a63c